datamade/how-to
# Recommendation of adoption: Heroku

## Overview

Collects our research on containerized deployment providers, and adds a recommendation of adoption for Heroku.

Connects #19.

## Testing Instructions

* View the diff on GitHub
* Confirm the markdown looks good
